### ISSUE TITLE
Removing KEY_KPENTER from codebase

### ISSUE THREAD

**Issue by CRImier** [ZeroPhone/ZPUI#37]
Earlier, when pyLCI was largely based on evdev and the HID subsystem (and used it to pass events), it mainly worked with USB numpads. The numpads I used didn't have KEY_ENTER, only KEY_KPENTER, so it was left in parallel in a lot of places like menus (not the best choice, but I didn't have enough programming experience back then). What's needed:
- [x] Add a "mapping" feature to "hid" input driver, allowing it to map keys to other keys - and hardcode "KEY_KPENTER":"KEY_ENTER" as a default
  - [x] The mapping has to be settable from `config.json`
- [x] Remove KEY_KPENTER from the code&docs, replacing it with KEY_ENTER everywhere it doesn't belong
- [x] Test it with real hardware

**Comment by CRImier** [comment]
KPENTER removal merged into `devel`, needs to be tested with a USB numpad/keyboard

**Comment by bolens1112** [comment]
#91 Completed removing KEY_KPENTER from the code&docs under ZeroPhone/ZPUI.
Added KEY_ENTER where needed.

**Comment by CRImier** [comment]
Thank you =)

**Comment by CRImier** [comment]
Done!
